### ISSUE TITLE
,/js/ added to the end of javascript

### ISSUE THREAD

**Issue by caevv** [fooman/speedster#8]
Hello,

On product page for some reason, a code of ,/js/ is added to the end of the line, breaking the JS files:

```
<script type="text/javascript" src="http://54.173.54.183:8080/skin/m/1421057512/js/prototype/prototype.js,/js/lib/ccard.js,/js/prototype/validation.js,/js/prototype/window.js,/js/scriptaculous/builder.js,/js/scriptaculous/effects.js,/js/scriptaculous/dragdrop.js,/js/scriptaculous/controls.js,/js/scriptaculous/slider.js,/js/varien/js.js,/js/varien/form.js,/js/varien/menu.js,/js/mage/translate.js,/js/mage/cookies.js,/js/class.horinaja.scriptaculous.pro.js,/js/GoogleAnalyticsPlus/distilled.FirstTouch.js,/js/mirasvit/core/jquery.min.js,/js/mirasvit/core/underscore.js,/js/mirasvit/core/backbone.js,/js/mirasvit/code/searchautocomplete/form.js,/js/mirasvit/code/searchautocomplete/autocomplete.js,/js/varien/product.js,/js/varien/configurable.js,/js/stereotabs.js,/js/lightbox.js,/js/calendar/calendar.js,/js/calendar/calendar-setup.js,/js/" ></script>
```

**Comment by caevv** [comment]
Config Version: 3.0.14
DB Version: 3.0.14
Data Version: 3.0.14
Step 1 - OK
Step 2 - OK
Step 3 - OK
Step 4 - OK
Starting Fooman_Speedster_Model_Selftester
Default store loaded
Magento version: 1.8.1.0
Speedster DB version: Not installed
Speedster Config version: 3.0.14
Checking file locations
Checking rewrites
Checking database
Result: success
Self-test finished

**Comment by chikoi** [comment]
Hi there,

Unfortunately, I could not replicate this error. For troubleshooting, please:

* Ensure that your store does not already contains minified files
* Ensure that your store does not contain significant Javascript customisation work has been done on your store and the added non-default Javascript files do not follow best practice Javascript (you can test with [JSLint](http://jslint.com/)
* Follow the installation instructions in [The Ultimate Guide to Installing Magento Extensions](http://cdn.fooman.co.nz/media/custom/upload/TheUltimateGuidetoInstallingMagentoExtensions.pdf)
* Follow all troubleshooting steps in the [Fooman Speedster Instructions and Troubleshooting Guide](http://cdn.fooman.co.nz/media/custom/upload/InstructionsandTroubleshooting-FoomanSpeedster.pdf)
* Ensure that you are not running any conflicting extensions, as this can conflict with CSS

This has a high success rate and usually solves 95% of issues we receive. If an error still persists, please contact your regular Magento developer or uninstall Fooman Speedster.

**Comment by fooman** [comment]
Should be solved with #10
